scripts/run_validator.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-branch=v3
-docker pull onsdigital/eq-schema-validator:$branch
-docker run -d -p 5001:5000 "onsdigital/eq-schema-validator:$branch"
\ No newline at end of file
+branch=eq-create-questionnaire-validator
```

Comment:
Update to 'latest' once https://github.com/ONSdigital/eq-questionnaire-validator/pull/1 is merged.